packages/app/src/app/components/CreateSandbox/Import/Import.tsx

```diff
@@ -87,6 +89,7 @@ export const Import: React.FC<ImportProps> = ({ onRepoSelect }) => {
       setShouldFetch(false);
 
       if (repo.private && isFree) {
+        // is this still possible?
```

Comment:
I think so, if a team member tries to import a private repo into a free workspace, it should be caught here.